doc/man5/x509v3_config.pod

```diff
@@ -6,112 +6,143 @@ x509v3_config - X509 V3 certificate extension configuration format
 
 =head1 DESCRIPTION
 
-Several of the OpenSSL utilities can add extensions to a certificate or
+Several OpenSSL commands can add extensions to a certificate or
 certificate request based on the contents of a configuration file.
+The syntax of this file is described in L<config(5)>.
+The commands typicaly have an option to specify the name of the configuration
+file, and a section within that file; see the documentation of the
+individual command for details.
 
-Typically the application will contain an option to point to an extension
-section. Each line of the extension section takes the form:
+This page uses B<extensions> as the name of the section, when needed
+in examples.
 
- extension_name=[critical,] extension_options
+Each entry in the extension section takes the form:
 
-If B<critical> is present then the extension will be critical.
+ name = [critical, ]value(s)
 
-The format of B<extension_options> depends on the value of B<extension_name>.
+If B<critical> is present then the extension will be marked as critical.
 
+The format of B<values> depends on the value of B<name>, many have a
+type-value pairing where the type and value are separated by a colon.
 There are four main types of extension: I<string> extensions, I<multi-valued>
-extensions, I<raw> and I<arbitrary> extensions.
+extensions, I<raw> extensions, and I<arbitrary> extensions.
+Each is described in the following paragraphs.
```

Comment:
Maybe itemize them?

Reply:
i don't want to touch this since at already got approved :)  but if you feel strongly, @slontis, I'll make it like this:
```types of extension:
 string
 multi-valued
 raw
 arbitrary
```

That is, I won't use ```=item```  I did this in other pages (d2i, exdata, etc); item takes up too much space IMO.

let me know what you think.

Reply:
comment deleted; additional commit pushed.